2021-09-27 21:32:53,802 INFO    MainThread:15149 [wandb_setup.py:_flush():69] setting env: {}
2021-09-27 21:32:53,802 INFO    MainThread:15149 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-09-27 21:32:53,802 INFO    MainThread:15149 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-09-27 21:32:53,803 INFO    MainThread:15149 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-27 21:32:53,803 INFO    MainThread:15149 [wandb_init.py:_log_setup():348] Logging user logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20210927_213253-22bhsms0/logs/debug.log
2021-09-27 21:32:53,803 INFO    MainThread:15149 [wandb_init.py:_log_setup():349] Logging internal logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20210927_213253-22bhsms0/logs/debug-internal.log
2021-09-27 21:32:53,803 INFO    MainThread:15149 [wandb_init.py:init():381] calling init triggers
2021-09-27 21:32:53,803 INFO    MainThread:15149 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/nerus_lenta/news_ner_dataset.csv', 'checkpoint_path': '/root/konst/model_checkpoints/ner_employment', 'model_path': '', 'batch_size': 128, 'epochs': 100, 'start_epoch': 0, 'test_size': 0.05, 'lam': 0, 'patience': 2, 'factor': 0.5, 'use_wandb': True, 'lr': 0.001, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 4}
2021-09-27 21:32:53,804 INFO    MainThread:15149 [wandb_init.py:init():430] starting backend
2021-09-27 21:32:53,804 INFO    MainThread:15149 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-27 21:32:53,817 INFO    MainThread:15149 [backend.py:ensure_launched():135] starting backend process...
2021-09-27 21:32:53,824 INFO    MainThread:15149 [backend.py:ensure_launched():140] started backend process with pid: 15167
2021-09-27 21:32:53,825 INFO    MainThread:15149 [wandb_init.py:init():435] backend started and connected
2021-09-27 21:32:53,829 INFO    MainThread:15149 [wandb_init.py:init():494] updated telemetry
2021-09-27 21:32:53,829 INFO    MainThread:15149 [wandb_init.py:init():517] communicating current version
2021-09-27 21:32:54,187 INFO    MainThread:15149 [wandb_init.py:init():522] got version response 
2021-09-27 21:32:54,187 INFO    MainThread:15149 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-27 21:32:54,471 INFO    MainThread:15149 [wandb_init.py:init():557] starting run threads in backend
2021-09-27 21:32:54,537 INFO    MainThread:15149 [wandb_run.py:_console_start():1605] atexit reg
2021-09-27 21:32:54,537 INFO    MainThread:15149 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-09-27 21:32:54,538 INFO    MainThread:15149 [wandb_run.py:_redirect():1484] Redirecting console.
2021-09-27 21:32:54,539 INFO    MainThread:15149 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-27 21:32:54,539 INFO    MainThread:15149 [wandb_init.py:init():582] run started, returning control to user process
2021-09-27 22:32:01,212 INFO    MainThread:15149 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 1
2021-09-27 22:32:01,228 INFO    MainThread:15149 [wandb_run.py:_restore():1547] restore
2021-09-27 22:32:03,398 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1094
  total_bytes: 1094
}

2021-09-27 22:32:03,596 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1094
  total_bytes: 1094
}

2021-09-27 22:32:04,380 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1094
  total_bytes: 7311
}

2021-09-27 22:32:04,482 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1094
  total_bytes: 7311
}

2021-09-27 22:32:04,583 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7311
  total_bytes: 7311
}

2021-09-27 22:32:04,684 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7311
  total_bytes: 7311
}

2021-09-27 22:32:04,785 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7311
  total_bytes: 7311
}

2021-09-27 22:32:04,886 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7311
  total_bytes: 7311
}

2021-09-27 22:32:04,988 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7311
  total_bytes: 7311
}

2021-09-27 22:32:05,090 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7311
  total_bytes: 7311
}

2021-09-27 22:32:05,235 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7311
  total_bytes: 7311
}

2021-09-27 22:32:05,634 INFO    MainThread:15149 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 7311
  total_bytes: 7311
}
local_info {
}

2021-09-27 22:32:06,715 INFO    MainThread:15149 [wandb_run.py:_show_files():1947] logging synced files
